requirements.txt

```diff
@@ -2,9 +2,10 @@ ansicolors==1.1.8
 docker==5.0.2
 h5py==2.7.1
 matplotlib==2.1.0
-numpy==1.13.3
+numpy==1.15.4
 pyyaml==5.4
 psutil==5.6.6
 scipy==1.0.0
 scikit-learn==0.19.1
 jinja2==2.10
+pandas==1.1.5
```

Comment:
Took me only 2.5 months to get back to this! It's now just using `csv` instead of `pandas`.

Reply:
> Took me only 2.5 months to get back to this! It's now just using `csv` instead of `pandas`.

@maumueller I would like to take this effort and include --json also as possible output format. Do you need help here to finish this feature?

Reply:
Sure, @filipecosta90. Let me merge this one and then you can make a clean PR from master.